styles/pages/rnd.css

```diff
@@ -0,0 +1,288 @@
+body.rnd main .section.image-before > .section-container {
+    position: relative;
+    max-width: 100%;
+  }
+  
+  body.rnd main .section.image-after > .section-container {
+    position: relative;
+    max-width: 100%;
+  }
+  
+  body.rnd main .section > .section-container img {
+      width: 100%;
+  }
+  
+  body.rnd main .section.image-before-after-t1 > div p img {
+    width: 402px;
+    height: 253px;
+    position: absolute;
+    top: -22vh;
+    z-index: 1;
+  }
+  
+  body.rnd main .section.image-before-after-t2 > div p img {
+    width: 181px;
+    height: 533px;
+    position: absolute;
+    top: -29vh;
+    z-index: 1;
+  }
+  
+  body.rnd main .section.image-before-after-t3 > div p img {
+    width: 190px;
+    height: 131px;
+    position: absolute;
+    top: -30vh;
+    z-index: 1;
+  }
+  
+  body.rnd main .section.image-before-after-t4 > div p img {
+    width: 107px;
+    height: 420px;
+    position: absolute;
+    top: -30vh;
+    z-index: 1;
+    right: 90px;
+  }
+
+  body.rnd main .section.image-before-after-t5 > div p img {
+    width: 392px;
+    height: 208;
+    position: absolute;
+    top: 6vh;
+    z-index: 1;
+    left: 54%;
+    transform: translate(-50%,-50%);
+  }
+  
+  body.rnd main .section.right > div p img {
+    right: 0;
+  }
+  
+  /* section - full-width */
+  body.rnd main .section.full-width > .section-container {
+    max-width: 100%;
+    padding-left: 0;
+    padding-right: 0;
+  }
+  
+  body.rnd main .section.with-background-image > .section-container {
+      max-width: 50%;
+      right: 8vw;
+  }
+  
+  /* section - dark indigo */
+  body.rnd main .section.bg-dark-indigo {
+      background-color: var(--underfill-dark-indigo);
+  }
+
+  body.rnd main .section > .section-container h1 {
+    font-size: var(--heading-font-size-xxl);
+  }
+
+  body.rnd main .section > .section-container h5, h4, h6 {
+    line-height: 1.8 !important;
```

Comment:
Can we avoid `!important` ?

Reply:
Yeah I did try to make that happen without Important. Actually other default columns css comes into action with line-height: 1.3rem without "Important"  - 

<img width="1523" alt="image" src="https://github.com/hlxsites/sunstar/assets/37831850/364be8b6-2473-492e-a553-f6d672e1621a">

Please advise.

Reply:
It's cause you're targeting

- body.rnd main .section > .section-container h5
- h4
- h6

Just repeat the beginning part as well `body.rnd main .section > .section-container` before h4, h6